packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -479,7 +479,8 @@ class IsolateManager {
   }
 
   void _sendToMessageBus(Event event) {
-    messageBus.addEvent(BusEvent(
+    // TODO(dantup): Some tests don't set up a message bus - is ? here reasonable?
+    messageBus?.addEvent(BusEvent(
```

Comment:
Is this a reasonable fix? Some tests don't set up a MessageBus and don't have easy access to the `testwidgetsWithContext` function (and are not widget tests).

Reply:
Can we mock this method instead? I think that would require us to make the method public

Reply:
This change has gone from this PR now during the rebase, as the `?` was already landed to fix another (or perhaps the same) issue in https://github.com/flutter/devtools/pull/2452.